bin/ember-template-lint.js

```diff
@@ -307,6 +308,14 @@ async function run() {
     return;
   }
 
+  if ((options.todoDaysToWarn || options.todoDaysToError) && !options.updateTodo) {
+    console.error(
+      'Using `--todo-days-to-warn` or `--todo-days-to-error` is only valid when the `--update-todo` option is being used.'
+    );
+    process.exitCode = 1;
+    return;
+  }
```

Comment:
This is great, do we need to write _basically_ the same thing as this for using `--update-todo --fix`?